prefect_docker/settings.py

```diff
@@ -0,0 +1,71 @@
+"""Module containing docker settings."""
+from typing import Any, Dict, Optional
+
+import docker
+from prefect.blocks.core import Block
+from pydantic import Field
+
+
+class DockerSettings(Block):
+    """
+    Block used to manage settings for a Docker server.
+
+    Attributes:
+        base_url: URL to the Docker server, e.g. `unix:///var/run/docker.sock`
+            or `tcp://127.0.0.1:1234`. If this is not set, the client will
+            be configured from environment variables.
+        version: The version of the API to use. Set to auto to
+            automatically detect the server's version.
+        timeout: Default timeout for API calls, in seconds.
+        max_pool_size: The maximum number of connections to save in the pool.
+        credstore_env: Override environment variables when calling the
+            credential store process.
+        client_kwargs: Additional keyword arguments to pass to
+            `docker.from_env()` or `DockerClient`.
+    """
+
+    _block_type_name = "Docker Settings"
+    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/2IfXXfMq66mrzJBDFFCHTp/6d8f320d9e4fc4393f045673d61ab612/Moby-logo.png?h=250"  # noqa
```

Comment:
I think we may want a seperate description since it's redundant to say that something is a block in the UI when we're on the Blocks screen.
```suggestion
    _block_type_name = "Docker Settings"
    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/2IfXXfMq66mrzJBDFFCHTp/6d8f320d9e4fc4393f045673d61ab612/Moby-logo.png?h=250"  # noqa
    _description = "Store settings for a interacting with a Docker host."
```